Resources/Prototypes/Entities/Objects/Specific/Medical/handheld_crew_monitor.yml

```diff
@@ -30,6 +30,21 @@
   - type: StaticPrice
     price: 500
 
+- type: entity
+  parent: HandheldCrewMonitor
+  id: HandheldCrewMonitorBorg
+  suffix: borg
+  components:
+  - type: HandheldCrewMonitor
```

Comment:
borgs cant insert cells

Reply:
If I did it right it should have a Microreactor cell in it that cent be ejected or switched

Reply:
why change chargeuse too

Reply:
I need to have the consumption increased to outpace the recharge of the Microreactor cell so it's not constantly kept open.